packages/ai-lab/src/lib/helpers.tsx

```diff
@@ -133,12 +134,27 @@ export async function getModelDetections(
   }
 }
 
-export async function getInferenceData(results: number[] | Detections) {
-  if (Array.isArray(results)) return results;
+export async function getInferenceData(
+  results: number[] | Detections,
+  modelConfig: ModelConfig = defaultModelConfig
+) {
+  // setting CLASSES as default label. Without it ssdInferData will not return any labels in the Inference data results.
+  const { labels = CLASSES } = modelConfig;
+
+  if (Array.isArray(results)) {
+    const res = results.map((data, key) => ({
+      class: key,
+      classLabel: `class ${key}`,
```

Comment:
why aren't we using labels here?

Reply:
I was under the impression that the classification models don't need labels and it was only an option for the SSD model that would need labels optionally.

Reply:
Nah, classification models def should have labels, too.   For instance NSFWJS has 5 classes, and 5 labels.

All detection models will use labels, and should apply the labels that fit that model.